plugins/repository-s3/src/main/java/org/opensearch/repositories/s3/S3ClientSettings.java

```diff
@@ -204,7 +204,7 @@ final class S3ClientSettings {
     static final Setting.AffixSetting<String> REGION = Setting.affixKeySetting(
         PREFIX,
         "region",
-        key -> new Setting<>(key, "us-west-2", Function.identity(), Property.NodeScope)
+        key -> new Setting<>(key, "", Function.identity(), Property.NodeScope)
```

Comment:
Should we make it null instead, to make it more explicit and maybe fail-fast if not set

Reply:
Setting doesn't allow for this :(